src/applications/check-in/pre-check-in/pages/Introduction/index.jsx

```diff
@@ -1,5 +1,158 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import AppointmentBlock from '../../components/AppointmentBlock';
+import Footer from '../../components/Footer';
+import { focusElement } from 'platform/utilities/ui';
+import { format, add } from 'date-fns';
+import Modal from '@department-of-veterans-affairs/component-library/Modal';
+// @TODO Add routing to button once routing merged in. Remove appointments once mock API merged in. Resolve answers to questions for UX team. Add unit test for intro.
+const Introduction = () => {
+  useEffect(() => {
+    focusElement('h1');
+  }, []);
+  const [privacyActModalOpen, setPrivacyActModalOpen] = useState(false);
+  const appointments = [
+    {
+      facility: 'LOMA LINDA VA CLINIC',
+      clinicPhoneNumber: '5551234567',
+      clinicFriendlyName: 'TEST CLINIC',
+      clinicName: 'LOM ACC CLINIC TEST',
+      appointmentIen: 'some-ien',
+      startTime: '2021-11-16T21:39:36',
+    },
+    {
+      facility: 'LOMA LINDA VA CLINIC',
+      clinicPhoneNumber: '5551234567',
+      clinicFriendlyName: 'TEST CLINIC 2',
+      clinicName: 'LOM ACC CLINIC TEST',
+      appointmentIen: 'some-ien',
+      startTime: '2021-11-16T23:00:00',
+    },
+  ];
+  const accordionContent = [
+    {
+      header: 'What happens after I answer the questions?',
+      body: (
+        <>
+          <p>
+            Changes to your contact information, insurance, next of kin, or
+            emergency contact will update your information across the VA.
+          </p>
+          <p>
+            If your provider requested additional questions, we’ll send your
+            answers to your provider through a secure electronic communication.
+            We’ll also ass the questionnaire to your medical record.
+          </p>
+          <p>
+            Your provider will review your answers and discuss them with you
+            during your appointment.
+          </p>
+        </>
+      ),
+    },
+    {
+      header: 'Will VA protect my personal health information?',
+      body: (
+        <>
+          <p>
+            We make every effort to keep your personal information private and
+            secure.
+          </p>
+          <a href="#">Read more about privacy and security on VA.gov</a>
+          <p>
+            You’re also responsible for protecting your personal health
+            information. If you print or download your information—or share it
+            electronically with others—you’ll need to take steps to protect it.
+          </p>
+          <a href="#">
+            Get tips for protecting your personal health information
+          </a>
+        </>
+      ),
+    },
+  ];
+  const appointmentsDateTime = new Date(appointments[0].startTime);
+  const StartButton = () => (
```

Comment:
We have an action link in the design library that we should be using 

https://design.va.gov/storybook/?path=/docs/components-action-link--primary

Reply:
Updated!